.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,39 @@
+### Directions
+_GitHub Issues are reserved for actionable bug reports and feature requests._
+_General questions should be sent to the influxdb@googlegroups.com mailing list._
+
+_Before opening an issue, search for similar bug reports or feature requests on GitHub Issues._
+_If no similar issue can be found, fill out either the "Bug Report" or the "Feature Request" section below.
+_Erase the other section and everything on and above this line._
+
+### Bug report
+
+__System info:__ [Include InfluxDB version, operating system name, and other relevant details]
+
+__Steps to reproduce:__
+
+1. [First Step]
+2. [Second Step]
+3. [and so on...]
+
+__Expected behavior:__ [What you expected to happen]
+
+__Actual behavior:__ [What actually happened]
+
+__Additional info:__ [Include gists of the config and logs if relevant]
```

Comment:
minor nit: gists can contain multiple files. Maybe change the comment to be "A single gist containing the relevant files, output, etc"